draft-add-requirements.md

```diff
@@ -104,92 +108,137 @@ the IETF may publish, such as DNS-over-QUIC {{?I-D.ietf-dprive-dnsoquic}}.
 
 Do53: Unencrypted DNS over UDP port 53, or TCP port 53 {{?RFC1035}}.
 
-Equivalent: See {{equivalence}}.
+Designated: See {{designation}}.
+
+Designator: The network or resolver that issued the designation.
 
 
-# Use case description
+# Overall use case
 
 It is often the case that a client possesses no specific configuration for how to operate DNS, and at some point
-joins a network that it has no previous knowledge about. In such a case the usual existing behaviour is to
-dynamically discover the network's recommended Do53 resolver and use it. This long-standing practice works in
-nearly all networks, but presents a number of privacy and security risks that were the motivation for the development
-of encrypted DNS.
+joins a network that it cannot authenticate. It may have no prior knowledge of the network, or it may have connected
+previously to a network that looked the same. In either case the usual behaviour, because of lack of specific
+configuration, is to dynamically discover the network's recommended Do53 resolver and use it. This long-standing practice
+works in nearly all networks, but presents a number of privacy and security risks that were the motivation for the
+development of encrypted DNS.
 
-The network's recommended unencrypted resolver may have a number of properties that differ from a generic resolver.
+The network's recommended Do53 resolver may have a number of properties that differ from a generic resolver.
 It may be able to answer names that are not known globally, it may exclude some names (for positive or negative
```

Comment:
Good catch. I second this; we should standardize on one term ("designated") throughout.

Reply:
I'm not sure I agree. We use designation to point to one or more ENCRYPTED resolvers. However this case is talking about DHCP/RA announcement of the Do53 using existing mechanisms. I'm fine with changing the word from "recommended" to something else, but if I change it to "designated" then I would also have to expand the definition to designation to include all types of existing announcement of Do53 resolvers.

Reply:
I think I would be fine with  "expand the definition to designation to include all types of existing announcement of Do53 resolvers."

Reply:
> if I change it to "designated" then I would also have to expand the definition to designation to include all types of existing announcement of Do53 resolvers.

I'm ok with that. To me, they're the same (if a client has decided never to use encryption, it will see both scenarios the same way; just because a designation is encrypted or not does nothing to change the fact that's it's optionally actionable information for the client to know).

I'm also ok with delineating so long as we're clear about the difference up front in a terms definition (that they're identical except one is for unencrypted resolvers and one is for encrypted resolvers).

Reply:
Ok, I've now expanded the definition to include Do53.